plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandler.java

```diff
@@ -38,26 +40,33 @@
 
   @Override
   public Object execute(ExecutionEvent event) throws ExecutionException {
+    Shell shell = HandlerUtil.getActiveShellChecked(event);
     try {
       IProject project = ProjectFromSelectionHelper.getProject(event);
       if (project == null) {
         throw new NullPointerException("Convert menu enabled for non-project resources");
       }
 
+      // check if Cloud SDK is available, prompts to configure if not
+      if (CloudSdkPrompter.getCloudSdkLocation(shell) == null) {
```

Comment:
Given that our tests is done with the Cloud SDK always available, and most of our user testing is done in such conditions too, I think it's safer to do this test first and exit early.  There is some part of our facet delegates that assumes the Cloud SDK.